cvcl/views/courseUser.py

```diff
@@ -0,0 +1,25 @@
+from django.shortcuts import render
```

Comment:
Seriously? importing one thing twice?
Go through the imports and remove the unused ones!